lib/index.js

```diff
@@ -1,26 +1,19 @@
 var $ = require('jquery');
 
 $(function() {
-  //when user clicks ABOUT, menu should expand to reveal about-menu
+  // when user clicks ABOUT, menu should expand to reveal about-menu
 
-  //use jquery to select the about button
+  // use jquery to select the about button
   let $aboutButton = $('[data-js="dropdown-about"]');
 
-  //select list items that make up the about-menu
+  // select list items that make up the about-menu
   let $aboutMenu = $('[data-js="about-menu"]');
 
-  //when about button is clicked, change orange, and display additional menu options*/
+  // when about button is clicked, change orange, and display additional menu options*/
   $aboutButton.on("click", function(){
 
-    //when about button is clicked, toggle class menu__item--aboutmenu
+    // when about button is clicked, toggle class menu__item--aboutmenu
     $aboutButton.toggleClass("menu__item--aboutmenu");
-
-    //if aboutButton has class menu__item--aboutmenu, change color to orange
-    if($aboutButton.hasClass("menu__item--aboutmenu")) {
```

Comment:
Removed this code... not because it's horrible, but rather in combination of the changes we went through in the HTML... it allowed for less lines of JS